python/instrumentation/openinference-instrumentation-boto/pyproject.toml

```
@@ -0,0 +1,57 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "openinference-instrumentation-boto"
```

should this be called bedrock instead of boto?
```suggestion
name = "openinference-instrumentation-bedrock"
```

---

That's a good question, we are instrumenting `bedrock` specifically but only for calls using the boto client. I'm leaning towards thinking `bedrock` is better too, let me think about this a little bit